src/Analyzers/Core/Analyzers/RemoveUnusedMembers/AbstractRemoveUnusedMembersDiagnosticAnalyzer.cs

```diff
@@ -24,6 +24,8 @@ internal abstract class AbstractRemoveUnusedMembersDiagnosticAnalyzer<TDocumenta
         where TDocumentationCommentTriviaSyntax : SyntaxNode
         where TIdentifierNameSyntax : SyntaxNode
     {
+        private static readonly SymbolDisplayFormat NameOnlyFormat = new();
```

Comment:
❔ Why do we not want to use `CSharpShortErrorMessageFormat` and `VisualBasicShortErrorMessageFormat` here?

Reply:
Generally speaking, they're just far too long.  Including things like full parameter lists.  These are needed by the compiler, since they need to often disambiguate what symbol is actually being refer'ed to.  That's not an issue here where we have the diagnostic right where we need it.

Reply:
What about using Name+ContainingType, so it doesn't need to be manually concatenated?

Reply:
yes.  that would make much more sense :)